docs/src/code/docs-style-guide.adoc

```diff
@@ -411,157 +493,196 @@ Follow these guidelines for punctuation:
 
-It's not clear whether the user should include the period in the string.
+Backticks are more precise than quotes and avoid painful escaping when input
+text contains quotes.
 
 === Inline code
 
-Inline code style is applied inline with regular text. Use inline code style:
+Inline code style is applied inline with regular text.
+
+Use inline code style:
 
-* For filenames or fragments of configuration files. For example, `.gitlab-ci.yml`, `CODEOWNERS`, and `only: [main]`.
-* For HTTP methods (`HTTP POST`) and HTTP status codes, both full (`404 File Not Found`) and abbreviated (`404`).
-For example: Send a `DELETE` request to delete the runner. Send a `POST` request to create one.
+* For filenames or fragments of configuration files. +
+  For example: `.gitlab-ci.yml`, `CODEOWNERS`, and `only: [main]`.
+* For methods and STATUS messages
+  For example: FIXME
 
-To apply inline code style, wrap the text in a single backtick (```). For example, `this is inline code style`.
+To apply inline code style, wrap the text in a single backtick (```). +
+For example: `this is inline code style`.
 
 === Code blocks
 
-Code block style separates code text from regular text. Use code block style for commands run in the command-line
-interface. Code block style is easier to copy and paste in a user's terminal window.
+Code block style separates code text from regular text.
 
-To apply code block style, wrap the text in triple backticks (three ```) and add a syntax highlighting hint. For
-example:
+Use code block style for commands run in the command-line interface. +
+Code block style is easier to copy and paste in a user's terminal window.
 
-`plaintext
-```plaintext
+// FIXME Code blocks styling
+//   https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-blocks/
+//   https://asciidoc-py.github.io/userguide.html#X104
+To apply code block style, wrap the text in quadruple dashes (four `-`)
+and add a `source` style with the adequate language.
+For example:
+
+
+----
 This is codeblock style
-```
-`
+----
 
 When using code block style:
 
-* Use quadruple backticks (four ```) to apply code block style when the code block you are styling has triple
-backticks in it. For example, when illustrating code block style.
+* Use quadruple backticks (four ```) to apply code block style when the
+  code block you are styling has triple backticks in it. +
+  For example, when illustrating code block style.
 * Add a blank line above and below code blocks.
-* Syntax highlight hints are required for code blocks. See the
-https://github.com/rouge-ruby/rouge/wiki/List-of-supported-languages-and-lexers[list of supported languages and lexers]
-for available syntax highlighters. Use `plaintext` if no better hint is available.
+* Syntax highlight hints are required for code blocks. +
+  See the https://github.com/rouge-ruby/rouge/wiki/List-of-supported-languages-and-lexers[list of supported languages and lexers]
```

Comment:
I think that this is different for Asciidoc. 
Note that Rust works better than C for comp files as it handles trple-quoted strings.